packages/web/src/common/store/pages/signon/sagas.ts

```diff
@@ -1105,7 +1059,8 @@ function* signIn(action: ReturnType<typeof signOnActions.signIn>) {
     const reportToSentry = yield* getContext('reportToSentry')
     reportToSentry({
       error: err,
-      name: 'Sign In: unknown error'
+      name: 'Sign In: unknown error',
```

Comment:
probably this should be fatal?
can't sign in is pretty bad

Reply:
I'm gonna do another pass tomorrow to start categorizing fatal errors - mainly I want to make sure fatal errors represent bad UI behavior (gonna test out throwing errors around)